.babelrc.js

```diff
@@ -9,11 +9,10 @@ if (process.env.NODE_ENV === 'production') {
 module.exports = {
     plugins,
     presets: [
-        process.env.BABEL_ENV === 'esm' ? [
-            'es2015', {
-                modules: false,
-            }
-        ] : 'es2015',
+        ['es2015', {
```

Comment:
I think it’s deprecated for a long time now and you need to use preset-env instead.

Reply:
Haha yeah, I've been lazy